Earthfile

```diff
@@ -25,6 +25,8 @@ lint:
 
 # build builds Crossplane for your native OS and architecture.
 build:
+  ARG USERPLATFORM
```

Comment:
this looks correct to me with my testing of `early +build` now (i get `darwin_arm64` binaries):
```
❯ tree _output
_output
├── bin
│   └── darwin_arm64
│       ├── crank
│       └── crossplane
└── charts
    └── crossplane-0.0.0-1717875335-2e37d912.tgz

3 directories, 3 files
```